qa/rolling-upgrade/build.gradle

```diff
@@ -18,71 +18,89 @@
  */
 
 import org.elasticsearch.gradle.test.RestIntegTestTask
+import org.elasticsearch.gradle.Version
 
 apply plugin: 'elasticsearch.standalone-test'
 
-task oldClusterTest(type: RestIntegTestTask) {
-  mustRunAfter(precommit)
+// dummy task for all bwc tests
```

Comment:
can you make this commetn more verbose why we do this and what the implications are?